src/bin/bat/main.rs

```diff
@@ -200,19 +200,19 @@ pub fn list_themes(cfg: &Config) -> Result<()> {
                 .ok();
             writeln!(stdout)?;
         }
+        writeln!(
+            stdout,
+            "Further themes can be installed to '{}', \
+        and are added to the cache with `bat cache --build`. \
```

Comment:
I guess this should be indented an extra level, so everything is the same level like it was before

Reply:
oh, thought i had executed `:Format` - will have a look at it

Reply:
I could imagine `cargo fmt` doesn't fix that because it's a multiline string

Reply:
ah ok - fixed it